### ISSUE TITLE
Reverse Mission Control Click and Drag [enhancement]

### ISSUE THREAD

**Issue by XInflict** [noah-nuebling/Mac-Mouse-Fix#541]
## Description
I'd like to use a counter intuitive drag to the mission control command, just like the default reverse scrolling on MacOS.

## Rationale
Faster Workflow, and move the screens based on muscle memory

## Additional info
–

**Comment by kecinzer** [comment]
This was a option in latest version, but in 3.0.0 beta there is no such option. Please add it as well.

**Comment by cihadturhan-st** [comment]
I was using v2 with natural direction and upgraded to 3.x.x and now the direction of left and right were reversed. 
Is there any option to change?

**Comment by cihadturhan** [comment]
I've checked the code in 15 mins or so.

@noah-nuebling I guess [that's the line](https://github.com/noah-nuebling/mac-mouse-fix/blob/master/Helper/InputTransformation/Drag/ModifiedDrag.m#L291) responsible horizontal drag for mission control.

I think removing minus sign will do what we like. I'm going to give a try if I can successfully build :)

**Comment by gamichalski** [comment]
Good @cihadturhan, please let us know about it.

I also would like to have this fixed, but I'm not sure how to build it.

**Comment by noah-nuebling** [comment]
Hey guys! The behaviour in 3.0.0 Beta 6 is because of [this line](https://github.com/noah-nuebling/mac-mouse-fix/blob/9800f21b27d88fd70ea1f24d6b33b951aae1933b/Helper/Core/Drag/ModifiedDrag.m#L330).

The code is already deactivated in the latest version of the source code, so for the next Beta version as well as the final 3.0.0 release, the Mission Control Click and Drag direction will be as in Mac Mouse Fix 2.

As for when the next Beta or final 3.0.0 release will come, unfortunately I can't say at this point. I would love to release 3.0.0 as soon as possible, and I hope to be able to do it this summer. But at the moment I can't work on MMF because I'm struggling a lot with my mental health.

**Comment by gamichalski** [comment]
@noah-nuebling I hope you feel better soon

I imagine you're going through a tough time, I wish you all the best.
Please let us know about it, this thing will be good for all of us.

**Comment by cihadturhan** [comment]
> But at the moment I can't work on MMF because I'm struggling a lot with my mental health.

Sorry to hear that Noah! Hope you feel better soon. MMF works fine in the current version so no rush in that :)
Take care and rest a lot.

**Comment by paulrouget** [comment]
@noah-nuebling Is it just a matter of doing a build and publishing a beta7 ? Can I help?

**Comment by gamichalski** [comment]
I also would like to help, but I don't know how to build the project, will be really good to have this released

**Comment by noah-nuebling** [comment]
Thanks for the kind words guys!

Publishing a fix for this is just a matter of cleaning up any changes I've been working on, writing release notes and then running some scripts.

But I always feel like it's not worth it to spend time on publishing a Beta when there's nothing new I want feedback on because it only further delays the final release, and I already have trouble focusing on that. I'm sorry about that.

However I think I know a workaround for this bug now.

The problem is that Mac Mouse Fix 3.0.0 Beta 6 tries to mirror the "Natural scrolling" preference of your Trackpad. If you go to System Settings > Trackpad > Scroll & Zoom and toggle the "Natural scrolling" setting, that will also invert the Click and Drag direction in Mac Mouse Fix.

However, if you've never toggled the "Natural scrolling" setting in System Settings, then Mac Mouse Fix can't read the value for the setting, and it falls back to assuming that natural scrolling is turned **off** (even though it should fall back to **on**, since the default setting for natural scrolling is **on**)

**Workaround**

To fix this, you can go to System Settings > Trackpad > Scroll & Zoom and toggle the "Natural scrolling" setting. Toggling this setting will invert the direction of click and drag in Mac Mouse Fix.

If the Trackpad settings aren't available on your system, you can also use the following terminal commands to toggle the Click and Drag inversion in Mac Mouse Fix:

To set Click and Drag direction to inverted:
```
defaults write -globalDomain com.apple.swipescrolldirection 0
```

To set Click and Drag direction to normal:
```
defaults write -globalDomain com.apple.swipescrolldirection 1
```

I hope this helps and is at least somewhat satisfactory. Let me know if you have more questions.

**Comment by gamichalski** [comment]
Thanks man, this helped me a lot

**Comment by noah-nuebling** [comment]
Hey guys, thanks again for chiming in! The drag direction should be fixed in [3.0.0 Beta 7](https://github.com/noah-nuebling/mac-mouse-fix/releases/tag/3.0.0-Beta-7). If you have any more questions, let me know.

**Comment by noah-nuebling** [comment]
> I also would like to help, but I don't know how to build the project, will be really good to have this released

Hey @gamichalski thank you very much for the offer, if there's something specific you would like to work on, shoot me an email and we can discuss.
